web/stats/dlc_structures.php

```diff
@@ -342,9 +339,9 @@
       "los": "20",
       "hp": "2200",
       "ra": "7",
-      "at": "6/arrow",
+      "at": "6+ 3x 1",
```

Comment:
This one is tricky indeed. It's +13 against ships for the first arrow because the thallasocracy tech doesn't just upgrade to harbor, it also includes an extra +4 against ships. Don't ask me why.
The extra arrows will always do 1 damage - The bonus damage doesn't matter here because it doesn't go above 1, which is the minimum damage for an attack anyway (and the minimum damage is applied after all other damage calculations).